pyston/docker/build_docker.sh

```diff
@@ -1,18 +1,22 @@
 #!/bin/sh
+#
+# Requires:
+# - docker buildx
+# - installed qemu emulation for the non native arch
+#   e.g.: docker run --privileged --rm tonistiigi/binfmt --install arm64
```

Comment:
Can you add these requirements to the readme (section "Extra dependencies for producing Pyston debian packages and portable directory release")?

Reply:
good point - documented it